descriptive-link

```diff
@@ -0,0 +1,180 @@
+---
+id: 
```

Comment:
I usually use https://www.browserling.com/tools/random-hex for generating the id